event.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Consider adding a `schema` or `$schema` property to the JSON object to specify the JSON Schema version being used. This can help with validation and documentation.